async-inference/HuggingFace-Async-Inference-Walkthrough.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.                "S3OutputPath": f"s3://twittersentimentanalysisoutput/output",
</pre><p>Don't hard-code S3 bucket that the customer won't own.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2910/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Hard coding removed